### PR TITLE
Feature21 and Feature 22 - Fit Score Calculations

### PR DESCRIPTION
**Pull request by arikamat**

### CODE REVIEW

**Comment by arikamat** [comment]
[Task 21: Develop Algorithms to Compare Resume Content with Job Description Requirements](https://trello.com/c/fBWKAIRq/25-task-21-develop-algorithms-to-compare-resume-content-with-job-description-requirements)

**Comment by arikamat** [comment]
[Task 22: Create Functions to Calculate a Fit Score Based on Keywords, Skills, and Experience](https://trello.com/c/EySGdbRZ/26-task-22-create-functions-to-calculate-a-fit-score-based-on-keywords-skills-and-experience)

File: tests/backend/test_utils_fit_score.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+Posted twice in 2021.
+Photography Portfolio (Unfinished)
+
+Took random pictures on my phone to post on social media.
+Certifications
+None
+Professional Development
+Attended one-day seminar on "Effective Communication in the Workplace" in 2019.
+"""
+def test_calculate_fit_score_normal():
```
Comment (haitham-awad) [inline tests/backend/test_utils_fit_score.py:143]: need to write documentation for all of these test functions

**Review by npaul64 — DISMISSED** [review]

File: tests/backend/test_utils_fit_score.py (hunk 1 above)
Comment (arikamat) [reply tests/backend/test_utils_fit_score.py:143]: fixed. TY

**Review by haitham-awad — APPROVED** [review]

**Review by npaul64 — APPROVED** [review]

**Pull request merged**